### ISSUE TITLE
Add method/API function to add a post type to zoninator

### ISSUE THREAD

**Issue by simonwheatley** [Automattic/zoninator#12]
Currently the code to add support to a post_type to Zoninator seems to be as follows:

``` php
add_post_type_support( 'video', $GLOBALS[ 'zoninator' ]->zone_taxonomy );
register_taxonomy_for_object_type( $GLOBALS[ 'zoninator' ]->zone_taxonomy, 'video' );

// Clear Zoninator supported post types cache
unset( $GLOBALS[ 'zoninator' ]->post_types );
```

Would be good to have a simpler API method or function.

Related: #2

**Comment by sboisvert** [comment]
@brettshumaker Can you take a look?

**Comment by brettshumaker** [comment]
@sboisvert 

Created a `register_zone_post_type()` method in this commit 94ab6cbcae2f207cc0914c245ee21d78677ff815

It's basically a wrapper for the above code that accepts a post type slug string or an array of post type slugs.

Also, #30 appears to be written against an older version of the plugin (0.6) and can likely be closed.

**Comment by brettshumaker** [comment]
@sboisvert Addressed your concerns with the 94ab6cb (dfb81cf and d557f52), then also addressed a few code formatting fixes in 75e637101d4dcfc30df3cf932bae65cb5f4809aa.

**Comment by sboisvert** [comment]
lgtm let's make a PR for this one

**Comment by sboisvert** [comment]
fixed in #73
